profiles/Ankit_Rai.md

```diff
@@ -0,0 +1,32 @@
+## Ankit Rai
+a high functioning geek, et cetera!
+![hey, it's Ankit!](https://pbs.twimg.com/profile_images/903953844542988288/XqKVmDxA_400x400.jpg)
+
+#### Get-in-Touch
+- web : [Ankit Rai](http://ankitrai.xyz)
+- mail: [abc@ankitrai.xyz](mailto:abc@ankitrai.xyz)
+
+#### Projects
+- [Power-Pi](https://github.com/ankitrai96/power-pi)
+- [multilingual Algorithm Vault](https://github.com/ankitrai96/multilingual-algorithm-vault)
+- [more on my GitHub ;)](https://github.com/ankitrai96/)
+
+#### Personal Details
```

Comment:
@ankitrai96 Are you sure you want to share all these personal details?

Reply:
If you are fine with it then I will merge to master.

Reply:
Yeah, I'm okay with it @stevencopelan.

It just came to me at that moment. That's why I put it there. However, there's nothing sensitive or confidential about me in it.

Anyway, thanks. You can go ahead and merge.

Reply:
:+1: